.travis.yml

```diff
@@ -12,15 +12,13 @@ env:
   - PLATFORMIO_ENV=esp12e
   - PLATFORMIO_ENV=nodemcuv2
 
-before_install:
-  - nvm install 6
-
 install:
   # PlatformIO
   - pip install -U https://github.com/platformio/platformio-core/archive/develop.zip
-  # copy password file
   - platformio update
   - platformio lib install
+  # install node
+  - nvm install node
```

Comment:
I still don't understand why to deviate from the official examples.

Reply:
Because it is more clean and the official examples are made for node projects.